charts/harvester-node-disk-manager/templates/daemonset.yaml

```
@@ -31,6 +31,26 @@ spec:
         command:
         - node-disk-manager
         env:
+        {{- with .Values.vendorFilter }}
+        - name: NDM_VENDOR_FILTER
+          value: {{ . | join "," | quote }}
+        {{- end }}
+        {{- with .Values.pathFilter }}
+        - name: NDM_PATH_FILTER
+          value: {{ . | join "," | quote }}
+        {{- end }}
+        {{- with .Values.labelFilter }}
+        - name: NDM_LABEL_FILTER
+          value: {{ . | join "," | quote }}
+        {{- end }}
+        {{- with .Values.rescanInterval }}
+        - name: NDM_RESCAN_INTERVAL
+          value: {{ . }}
```

```suggestion
          value: {{ . | quote }}
```

The env values of containers need to be in string type. If a user enters value 30, it will become an integer value.